infra/feast-operator/config/overlays/odh/params.env

```diff
@@ -0,0 +1 @@
+odh-feast-operator-controller-image=feastdev/feast-operator:0.42.0
```

Comment:
why is this necessary?

Reply:
to override the operator image.
one case as per my understanding adding into`relatedImages` section in the CSV to override the operator image with a SHA-based image, or use odh generated version of feast operator image. 

https://github.com/redhatHameed/opendatahub-operator/blob/featurestore-incubation/controllers/components/feastoperator/feastoperator_support.go#L25

https://github.com/redhatHameed/opendatahub-operator/blob/featurestore-incubation/controllers/components/feastoperator/feastoperator.go#L60

Reply:
related images changes are automated separately

Reply:
hmm... i see what you mean though

Reply:
can you add this as part of this PR?

Reply:
also, since this will mostly be deployed to openshift envs, maybe use the fully qualified image to avoid repo priority issues - 
```suggestion
odh-feast-operator-controller-image=docker.io/feastdev/feast-operator:0.42.0
```